.gitignore

```diff
@@ -5,6 +5,7 @@ vendor/
 .php_cs.cache
 doc/.couscous
 .subsplit
+.idea
```

Comment:
You should put this line in your global Git configuration (see https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration).